src/server/json_family.cc

```diff
@@ -490,8 +491,66 @@ OpResult<long> OpDel(const OpArgs& op_args, string_view key, string_view path) {
   return total_deletions;
 }
 
+OpResult<vector<StringVector>> OpObjKeys(const OpArgs& op_args, string_view key,
```

Comment:
maybe worth adding a comment explaining what `vector<StringVector>` corresponds to.

Reply:
after reading the code, I kinda understand the structure but it feels weird. 
the order of this vector is not well defined. Does `evaluate` preserves the original order?

Reply:
I did some checks and take a look at jsoncons codebase and what I can say is that evaluate preserves the hierarchy of the objects within the tree.